recipes/tktable/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set version = "2.10" %}
+
+package:
+  name: tktable
+  version: {{ version }}
+
+source:
+  url: http://downloads.sourceforge.net/project/tktable/tktable/{{ version }}/Tktable{{ version }}.tar.gz
+  sha256: c335117fa1be45fe4d3032e96fd4b4641fff6a4f8467878608dabed11198a4cb
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+    - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libxcb') }}                # [linux]
+    - {{ cdt('libxau') }}                # [linux]
+  host:
+    - tk
+  run:
+    - tk
+
+test:
+  commands:
+    - sudo yum install xorg-x11-server-Xvfb -y  && echo "if {[catch {package require -exact Tktable {{ version }}; exit 0}]} {exit 1}" | xvfb-run tclsh  # [linux]
```

Comment:
Use https://conda-forge.org/docs/maintainer/knowledge_base.html#yum-requirements-txt for this